packages/website/src/features/Deploy/QueueDrawer.tsx

```diff
@@ -380,12 +380,15 @@ export const QueuedTxns = ({
                           colorScheme="teal"
                           w="100%"
                           isDisabled={
+                            stager.signing ||
```

Comment:
in accordance with showing a reason any time a button is disabled, can we put some indication somewhere... somehow on the page that says "currently signing txn" to indicate the button is disabled (maybe the user doenst realize their signing window was opened when they click on the button)

Reply:
Nice catch, sure, I'll add it ASAP, thank you